server/service/hosts.go

```diff
@@ -532,6 +539,9 @@ func (svc *Service) GetHost(ctx context.Context, id uint, opts fleet.HostDetailO
 	if err != nil {
 		return nil, ctxerr.Wrap(ctx, err, "get host")
 	}
+	if !opts.IncludeCriticalVulnerabilitiesCount {
+		host.HostIssues.CriticalVulnerabilitiesCount = nil
+	}
```

Comment:
Same here right? (The join to `host_issues` is cheap enough that we don't need to prevent the join in `svc.ds.Host` when `opts.IncludeCriticalVulnerabilitiesCount=false`)

Reply:
This is different. For premium license, we don't return critical_vulnerabilities_count